apps/admin-gui/src/app/app.component.ts

```diff
@@ -46,6 +55,31 @@ export class AppComponent implements OnInit {
 
   ngOnInit(): void {
     this.principal = this.store.getPerunPrincipal();
+    this.loadAppVersion();
+    interval(30000).subscribe(() => {
+      this.loadAppVersion();
+    });
+  }
+
+  loadAppVersion() {
+    const httpHeaders = new HttpHeaders({
+      'Cache-Control': 'no-cache, no-store, must-revalidate, post-check=0, pre-check=0',
+      'Pragma': 'no-cache',
+      'Expires': '0'
+    });
+    this.http.get('/assets/config/version.json', { headers: httpHeaders })
+      .subscribe(result => {
+        const version = result['version'];
+        if (version) {
+          if (this.version && this.version !== version) {
```

Comment:
```suggestion
          if (this.version && version !== 'SNAPSHOT' && this.version !== version) {
```
Also, please add this change that would stop the alert from being shown in the development mode.